lib/OrdinaryDiffEqCore/src/interp_func.jl

```diff
@@ -59,3 +59,15 @@ function InterpolationData(id::InterpolationData, f)
         id.differential_vars,
         id.sensitivitymode)
 end
+
+# strip interpolation of function information
+function SciMLBase.strip_interpolation(id::InterpolationData)
+    InterpolationData(nothing, id.timeseries,
```

Comment:
Won't this need a validation that it's correct to strip?

Reply:
I'm not sure what you mean, is there a situation when calling this function where the function shouldn't be taken out?